2017-06-13 09:44:59,333 INFO TC56_kill_non_leader_mon.py [line:25] 
This test case will do the following steps:
1. start IO on 10 images with randwrite and verify
2. login the non-leader monitor, and kill the mon process
3. check cluster status, Io status and cluster quorum status
4. start the killed monitor
5. check cluster status, does the leader monitor will be back????

2017-06-13 09:45:00,042 INFO monitors.py [line:126]    "quorum_leader_name": "taheo125",
stdin: is not a tty

2017-06-13 09:45:00,042 INFO monitors.py [line:129]    "quorum_leader_name": "taheo125",
2017-06-13 09:45:02,047 INFO TC56_kill_non_leader_mon.py [line:30] 
Step1: Check IO from clients
2017-06-13 09:45:02,527 INFO client.py [line:172] ['enali    30609  30608  0 01:45 ?        00:00:00 bash -c sudo -i ps -ef | grep fio ', 'denali    30611  30609  0 01:45 ?        00:00:00 grep fio', 'root      33296      1  0 Jun12 ?        00:00:00 sudo -i fio -direct=1 -iodepth_batch_complete=1 -ioengine=rbd -clientname=client100 -pool=reliablityTestPool -rbdname=client100rbdImg1 -rw=randwrite -bs=8K -size=200G -norandommap=1 -randrepeat=0 -verify=md5 -verify_fatal=1 -verify_dump=1 -do_verify=1 -overwrite=1 -iodepth=256 -name=client100rbdImg1', 'root      33298  33296 68 Jun12 ?        01:22:07 fio -direct=1 -iodepth_batch_complete=1 -ioengine=rbd -clientname=client100 -pool=reliablityTestPool -rbdname=client100rbdImg1 -rw=randwrite -bs=8K -size=200G -norandommap=1 -randrepeat=0 -verify=md5 -verify_fatal=1 -verify_dump=1 -do_verify=1 -overwrite=1 -iodepth=256 -name=client100rbdImg1', 'root      33365      1  0 Jun12 ?        00:00:00 sudo -i fio -direct=1 -iodepth_batch_complete=1 -ioengine=rbd -clientname=client100 -pool=reliablityTestPool -rbdname=client100rbdImg2 -rw=randwrite -bs=8K -size=300G -norandommap=1 -randrepeat=0 -verify=md5 -verify_fatal=1 -verify_dump=1 -do_verify=1 -overwrite=1 -iodepth=256 -name=client100rbdImg2', 'root      33366  33365 85 Jun12 ?        01:42:31 fio -direct=1 -iodepth_batch_complete=1 -ioengine=rbd -clientname=client100 -pool=reliablityTestPool -rbdname=client100rbdImg2 -rw=randwrite -bs=8K -size=300G -norandommap=1 -randrepeat=0 -verify=md5 -verify_fatal=1 -verify_dump=1 -do_verify=1 -overwrite=1 -iodepth=256 -name=client100rbdImg2', 'root      33436      1  0 Jun12 ?        00:00:00 sudo -i fio -direct=1 -iodepth_batch_complete=1 -ioengine=rbd -clientname=client100 -pool=reliablityTestPool -rbdname=client100rbdImg3 -rw=randwrite -bs=8K -size=400G -norandommap=1 -randrepeat=0 -verify=md5 -verify_fatal=1 -verify_dump=1 -do_verify=1 -overwrite=1 -iodepth=256 -name=client100rbdImg3', 'root      33439  33436 99 Jun12 ?        02:02:25 fio -direct=1 -iodepth_batch_complete=1 -ioengine=rbd -clientname=client100 -pool=reliablityTestPool -rbdname=client100rbdImg3 -rw=randwrite -bs=8K -size=400G -norandommap=1 -randrepeat=0 -verify=md5 -verify_fatal=1 -verify_dump=1 -do_verify=1 -overwrite=1 -iodepth=256 -name=client100rbdImg3', 'root      33508      1  0 Jun12 ?        00:00:00 sudo -i fio -direct=1 -iodepth_batch_complete=1 -ioengine=rbd -clientname=client100 -pool=reliablityTestPool -rbdname=client100rbdImg4 -rw=randwrite -bs=8K -size=500G -norandommap=1 -randrepeat=0 -verify=md5 -verify_fatal=1 -verify_dump=1 -do_verify=1 -overwrite=1 -iodepth=256 -name=client100rbdImg4', 'root      33510  33508 99 Jun12 ?        02:25:44 fio -direct=1 -iodepth_batch_complete=1 -ioengine=rbd -clientname=client100 -pool=reliablityTestPool -rbdname=client100rbdImg4 -rw=randwrite -bs=8K -size=500G -norandommap=1 -randrepeat=0 -verify=md5 -verify_fatal=1 -verify_dump=1 -do_verify=1 -overwrite=1 -iodepth=256 -name=client100rbdImg4', 'stdin: is not a tty', '']
2017-06-13 09:45:02,527 INFO client.py [line:174] IO is running
2017-06-13 09:46:02,585 INFO monitors.py [line:55] mon is  taheo126
2017-06-13 09:46:02,585 INFO monitors.py [line:56] execute command is sudo -i stop ceph-mon id=taheo126 & sleep 5
2017-06-13 09:46:07,772 ERROR monitors.py [line:61] Error when shutdown mon taheo126
2017-06-13 09:46:07,772 ERROR monitors.py [line:62] sudo -i stop ceph-mon id=taheo126 & sleep 5
2017-06-13 09:46:07,772 ERROR monitors.py [line:63] tdin: is not a tty
stop: Unknown instance: ceph/taheo126

2017-06-13 09:46:07,772 INFO monitors.py [line:68] mon is  taheo126
2017-06-13 09:46:07,772 INFO monitors.py [line:69] execute command is sudo -i ceph-mon -i taheo126 & sleep 30
2017-06-13 09:46:37,989 ERROR monitors.py [line:75] Error when start mon taheo126
2017-06-13 09:46:37,989 ERROR monitors.py [line:76] sudo -i ceph-mon -i taheo126 & sleep 30
2017-06-13 09:46:37,989 ERROR monitors.py [line:77] tdin: is not a tty
monitor data directory at '/var/lib/ceph/mon/ceph-taheo126' does not exist: have you run 'mkfs'?

2017-06-13 09:46:37,989 INFO TC56_kill_non_leader_mon.py [line:41] 
Step2: kill non-leader mon 10 times
2017-06-13 09:46:38,238 INFO monitors.py [line:45] ['oot     12887     1  2 Jun12 ?        00:20:56 /usr/local/bin/ceph-mon --cluster=ceph -i tahoe126 -f --setuser root --setgroup root', 'denali   17487 17486  0 09:46 ?        00:00:00 bash -c ps -ef | grep ceph-mon', 'denali   17490 17487  0 09:46 ?        00:00:00 grep ceph-mon', '']
2017-06-13 09:46:38,238 INFO monitors.py [line:51] mon pid is 12887
2017-06-13 09:46:38,239 INFO monitors.py [line:92] execute command is sudo -i kill -9 12887 & sleep 3
2017-06-13 09:47:12,050 INFO monitors.py [line:126]    "quorum_leader_name": "taheo125",
stdin: is not a tty

2017-06-13 09:47:12,051 INFO monitors.py [line:129]    "quorum_leader_name": "taheo125",
2017-06-13 09:47:12,051 INFO TC56_kill_non_leader_mon.py [line:50] now deal the non-leader mon  taheo125
2017-06-13 09:47:12,051 INFO monitors.py [line:68] mon is  taheo126
2017-06-13 09:47:12,051 INFO monitors.py [line:69] execute command is sudo -i ceph-mon -i taheo126 & sleep 30
2017-06-13 09:47:42,268 ERROR monitors.py [line:75] Error when start mon taheo126
2017-06-13 09:47:42,268 ERROR monitors.py [line:76] sudo -i ceph-mon -i taheo126 & sleep 30
2017-06-13 09:47:42,268 ERROR monitors.py [line:77] tdin: is not a tty
monitor data directory at '/var/lib/ceph/mon/ceph-taheo126' does not exist: have you run 'mkfs'?

2017-06-13 09:47:42,268 INFO monitors.py [line:103] node is  taheo126
2017-06-13 09:47:42,269 INFO monitors.py [line:104] execute command is ps -ef | grep 'ceph-mon' 
2017-06-13 09:47:42,555 INFO monitors.py [line:106] oot     17513     1  3 09:46 ?        00:00:01 /usr/local/bin/ceph-mon --cluster=ceph -i tahoe126 -f --setuser root --setgroup root
denali   23674 23665  0 09:47 ?        00:00:00 bash -c ps -ef | grep 'ceph-mon' 
denali   23677 23674  0 09:47 ?        00:00:00 grep ceph-mon

2017-06-13 09:47:42,556 INFO monitors.py [line:114] taheo126 is alrady started
2017-06-13 09:48:12,586 INFO cluster.py [line:211] execute command is sudo -i ceph -s
2017-06-13 09:48:12,948 INFO cluster.py [line:213]    cluster c1b31c6e-ebc7-4c97-98ec-d59964f99e42
     health HEALTH_OK
     monmap e3: 3 mons at {taheo125=192.168.40.125:6789/0,tahoe126=192.168.40.126:6789/0,tahoe127=192.168.40.127:6789/0}
            election epoch 56, quorum 0,1,2 taheo125,tahoe126,tahoe127
     osdmap e429: 18 osds: 18 up, 18 in
            flags sortbitwise,require_jewel_osds,require_kraken_osds
      pgmap v50566: 3216 pgs, 13 pools, 1498 GB data, 375 kobjects
            2055 GB used, 4134 GB / 6189 GB avail
                3216 active+clean
  client io 145 MB/s rd, 18863 op/s rd, 0 op/s wr
stdin: is not a tty

2017-06-13 09:48:12,948 INFO cluster.py [line:238] PG number is 3216
2017-06-13 09:48:12,948 INFO cluster.py [line:239] usefull PG number is 3216
2017-06-13 09:48:12,948 INFO TC56_kill_non_leader_mon.py [line:67] stop mon service on taheo126 in cluster successfully
2017-06-13 09:48:13,532 INFO monitors.py [line:126]    "quorum_leader_name": "taheo125",
stdin: is not a tty

2017-06-13 09:48:13,533 INFO monitors.py [line:129]    "quorum_leader_name": "taheo125",
2017-06-13 09:48:13,533 INFO TC56_kill_non_leader_mon.py [line:83] now the leader mon is taheo125
2017-06-13 09:48:13,533 INFO TC56_kill_non_leader_mon.py [line:85] the leader mon is not impacted
2017-06-13 09:48:13,785 INFO client.py [line:172] ['oot      33365      1  0 Jun12 ?        00:00:00 sudo -i fio -direct=1 -iodepth_batch_complete=1 -ioengine=rbd -clientname=client100 -pool=reliablityTestPool -rbdname=client100rbdImg2 -rw=randwrite -bs=8K -size=300G -norandommap=1 -randrepeat=0 -verify=md5 -verify_fatal=1 -verify_dump=1 -do_verify=1 -overwrite=1 -iodepth=256 -name=client100rbdImg2', 'root      33366  33365 84 Jun12 ?        01:44:40 fio -direct=1 -iodepth_batch_complete=1 -ioengine=rbd -clientname=client100 -pool=reliablityTestPool -rbdname=client100rbdImg2 -rw=randwrite -bs=8K -size=300G -norandommap=1 -randrepeat=0 -verify=md5 -verify_fatal=1 -verify_dump=1 -do_verify=1 -overwrite=1 -iodepth=256 -name=client100rbdImg2', 'root      33436      1  0 Jun12 ?        00:00:00 sudo -i fio -direct=1 -iodepth_batch_complete=1 -ioengine=rbd -clientname=client100 -pool=reliablityTestPool -rbdname=client100rbdImg3 -rw=randwrite -bs=8K -size=400G -norandommap=1 -randrepeat=0 -verify=md5 -verify_fatal=1 -verify_dump=1 -do_verify=1 -overwrite=1 -iodepth=256 -name=client100rbdImg3', 'root      33439  33436 99 Jun12 ?        02:04:34 fio -direct=1 -iodepth_batch_complete=1 -ioengine=rbd -clientname=client100 -pool=reliablityTestPool -rbdname=client100rbdImg3 -rw=randwrite -bs=8K -size=400G -norandommap=1 -randrepeat=0 -verify=md5 -verify_fatal=1 -verify_dump=1 -do_verify=1 -overwrite=1 -iodepth=256 -name=client100rbdImg3', 'root      33508      1  0 Jun12 ?        00:00:00 sudo -i fio -direct=1 -iodepth_batch_complete=1 -ioengine=rbd -clientname=client100 -pool=reliablityTestPool -rbdname=client100rbdImg4 -rw=randwrite -bs=8K -size=500G -norandommap=1 -randrepeat=0 -verify=md5 -verify_fatal=1 -verify_dump=1 -do_verify=1 -overwrite=1 -iodepth=256 -name=client100rbdImg4', 'root      33510  33508 99 Jun12 ?        02:27:54 fio -direct=1 -iodepth_batch_complete=1 -ioengine=rbd -clientname=client100 -pool=reliablityTestPool -rbdname=client100rbdImg4 -rw=randwrite -bs=8K -size=500G -norandommap=1 -randrepeat=0 -verify=md5 -verify_fatal=1 -verify_dump=1 -do_verify=1 -overwrite=1 -iodepth=256 -name=client100rbdImg4', 'denali    41220  41219  0 01:48 ?        00:00:00 bash -c sudo -i ps -ef | grep fio ', 'denali    41222  41220  0 01:48 ?        00:00:00 grep fio', 'stdin: is not a tty', '']
2017-06-13 09:48:13,785 INFO client.py [line:174] IO is running
2017-06-13 09:48:14,033 INFO monitors.py [line:45] ['oot     17513     1  2 09:46 ?        00:00:02 /usr/local/bin/ceph-mon --cluster=ceph -i tahoe126 -f --setuser root --setgroup root', 'denali   26761 26754  0 09:48 ?        00:00:00 bash -c ps -ef | grep ceph-mon', 'denali   26764 26761  0 09:48 ?        00:00:00 grep ceph-mon', '']
2017-06-13 09:48:14,033 INFO monitors.py [line:51] mon pid is 17513
2017-06-13 09:48:14,033 INFO monitors.py [line:92] execute command is sudo -i kill -9 17513 & sleep 3
2017-06-13 09:48:47,848 INFO monitors.py [line:126]    "quorum_leader_name": "taheo125",
stdin: is not a tty

2017-06-13 09:48:47,849 INFO monitors.py [line:129]    "quorum_leader_name": "taheo125",
2017-06-13 09:48:47,849 INFO TC56_kill_non_leader_mon.py [line:50] now deal the non-leader mon  taheo125
2017-06-13 09:48:47,849 INFO monitors.py [line:68] mon is  taheo126
2017-06-13 09:48:47,849 INFO monitors.py [line:69] execute command is sudo -i ceph-mon -i taheo126 & sleep 30
2017-06-13 09:49:18,066 ERROR monitors.py [line:75] Error when start mon taheo126
2017-06-13 09:49:18,066 ERROR monitors.py [line:76] sudo -i ceph-mon -i taheo126 & sleep 30
2017-06-13 09:49:18,066 ERROR monitors.py [line:77] tdin: is not a tty
monitor data directory at '/var/lib/ceph/mon/ceph-taheo126' does not exist: have you run 'mkfs'?

2017-06-13 09:49:18,066 INFO monitors.py [line:103] node is  taheo126
2017-06-13 09:49:18,066 INFO monitors.py [line:104] execute command is ps -ef | grep 'ceph-mon' 
2017-06-13 09:49:18,284 INFO monitors.py [line:106] oot     26803     1  2 09:48 ?        00:00:01 /usr/local/bin/ceph-mon --cluster=ceph -i tahoe126 -f --setuser root --setgroup root
denali   32248 32247  0 09:49 ?        00:00:00 bash -c ps -ef | grep 'ceph-mon' 
denali   32250 32248  0 09:49 ?        00:00:00 grep ceph-mon

2017-06-13 09:49:18,284 INFO monitors.py [line:114] taheo126 is alrady started
2017-06-13 09:49:48,306 INFO cluster.py [line:211] execute command is sudo -i ceph -s
2017-06-13 09:49:48,718 INFO cluster.py [line:213]    cluster c1b31c6e-ebc7-4c97-98ec-d59964f99e42
     health HEALTH_OK
     monmap e3: 3 mons at {taheo125=192.168.40.125:6789/0,tahoe126=192.168.40.126:6789/0,tahoe127=192.168.40.127:6789/0}
            election epoch 58, quorum 0,1,2 taheo125,tahoe126,tahoe127
     osdmap e429: 18 osds: 18 up, 18 in
            flags sortbitwise,require_jewel_osds,require_kraken_osds
      pgmap v50601: 3216 pgs, 13 pools, 1498 GB data, 375 kobjects
            2055 GB used, 4134 GB / 6189 GB avail
                3216 active+clean
  client io 147 MB/s rd, 18986 op/s rd, 0 op/s wr
stdin: is not a tty

2017-06-13 09:49:48,718 INFO cluster.py [line:238] PG number is 3216
2017-06-13 09:49:48,718 INFO cluster.py [line:239] usefull PG number is 3216
2017-06-13 09:49:48,719 INFO TC56_kill_non_leader_mon.py [line:67] stop mon service on taheo126 in cluster successfully
2017-06-13 09:49:49,273 INFO monitors.py [line:126]    "quorum_leader_name": "taheo125",
stdin: is not a tty

2017-06-13 09:49:49,274 INFO monitors.py [line:129]    "quorum_leader_name": "taheo125",
2017-06-13 09:49:49,274 INFO TC56_kill_non_leader_mon.py [line:83] now the leader mon is taheo125
2017-06-13 09:49:49,274 INFO TC56_kill_non_leader_mon.py [line:85] the leader mon is not impacted
2017-06-13 09:49:49,491 INFO client.py [line:172] ['oot      33365      1  0 Jun12 ?        00:00:00 sudo -i fio -direct=1 -iodepth_batch_complete=1 -ioengine=rbd -clientname=client100 -pool=reliablityTestPool -rbdname=client100rbdImg2 -rw=randwrite -bs=8K -size=300G -norandommap=1 -randrepeat=0 -verify=md5 -verify_fatal=1 -verify_dump=1 -do_verify=1 -overwrite=1 -iodepth=256 -name=client100rbdImg2', 'root      33366  33365 84 Jun12 ?        01:45:47 fio -direct=1 -iodepth_batch_complete=1 -ioengine=rbd -clientname=client100 -pool=reliablityTestPool -rbdname=client100rbdImg2 -rw=randwrite -bs=8K -size=300G -norandommap=1 -randrepeat=0 -verify=md5 -verify_fatal=1 -verify_dump=1 -do_verify=1 -overwrite=1 -iodepth=256 -name=client100rbdImg2', 'root      33436      1  0 Jun12 ?        00:00:00 sudo -i fio -direct=1 -iodepth_batch_complete=1 -ioengine=rbd -clientname=client100 -pool=reliablityTestPool -rbdname=client100rbdImg3 -rw=randwrite -bs=8K -size=400G -norandommap=1 -randrepeat=0 -verify=md5 -verify_fatal=1 -verify_dump=1 -do_verify=1 -overwrite=1 -iodepth=256 -name=client100rbdImg3', 'root      33439  33436 99 Jun12 ?        02:05:40 fio -direct=1 -iodepth_batch_complete=1 -ioengine=rbd -clientname=client100 -pool=reliablityTestPool -rbdname=client100rbdImg3 -rw=randwrite -bs=8K -size=400G -norandommap=1 -randrepeat=0 -verify=md5 -verify_fatal=1 -verify_dump=1 -do_verify=1 -overwrite=1 -iodepth=256 -name=client100rbdImg3', 'root      33508      1  0 Jun12 ?        00:00:00 sudo -i fio -direct=1 -iodepth_batch_complete=1 -ioengine=rbd -clientname=client100 -pool=reliablityTestPool -rbdname=client100rbdImg4 -rw=randwrite -bs=8K -size=500G -norandommap=1 -randrepeat=0 -verify=md5 -verify_fatal=1 -verify_dump=1 -do_verify=1 -overwrite=1 -iodepth=256 -name=client100rbdImg4', 'root      33510  33508 99 Jun12 ?        02:29:01 fio -direct=1 -iodepth_batch_complete=1 -ioengine=rbd -clientname=client100 -pool=reliablityTestPool -rbdname=client100rbdImg4 -rw=randwrite -bs=8K -size=500G -norandommap=1 -randrepeat=0 -verify=md5 -verify_fatal=1 -verify_dump=1 -do_verify=1 -overwrite=1 -iodepth=256 -name=client100rbdImg4', 'denali    46555  46554  0 01:49 ?        00:00:00 bash -c sudo -i ps -ef | grep fio ', 'denali    46557  46555  0 01:49 ?        00:00:00 grep fio', 'stdin: is not a tty', '']
2017-06-13 09:49:49,491 INFO client.py [line:174] IO is running
2017-06-13 09:49:49,746 INFO monitors.py [line:45] ['oot     26803     1  2 09:48 ?        00:00:02 /usr/local/bin/ceph-mon --cluster=ceph -i tahoe126 -f --setuser root --setgroup root', 'denali   35057 35056  0 09:49 ?        00:00:00 bash -c ps -ef | grep ceph-mon', 'denali   35059 35057  0 09:49 ?        00:00:00 grep ceph-mon', '']
2017-06-13 09:49:49,746 INFO monitors.py [line:51] mon pid is 26803
2017-06-13 09:49:49,746 INFO monitors.py [line:92] execute command is sudo -i kill -9 26803 & sleep 3
2017-06-13 09:50:23,485 INFO monitors.py [line:126]    "quorum_leader_name": "taheo125",
stdin: is not a tty

2017-06-13 09:50:23,485 INFO monitors.py [line:129]    "quorum_leader_name": "taheo125",
2017-06-13 09:50:23,485 INFO TC56_kill_non_leader_mon.py [line:50] now deal the non-leader mon  taheo125
2017-06-13 09:50:23,485 INFO monitors.py [line:68] mon is  taheo126
2017-06-13 09:50:23,485 INFO monitors.py [line:69] execute command is sudo -i ceph-mon -i taheo126 & sleep 30
2017-06-13 09:50:53,670 ERROR monitors.py [line:75] Error when start mon taheo126
2017-06-13 09:50:53,670 ERROR monitors.py [line:76] sudo -i ceph-mon -i taheo126 & sleep 30
2017-06-13 09:50:53,670 ERROR monitors.py [line:77] tdin: is not a tty
monitor data directory at '/var/lib/ceph/mon/ceph-taheo126' does not exist: have you run 'mkfs'?

2017-06-13 09:50:53,670 INFO monitors.py [line:103] node is  taheo126
2017-06-13 09:50:53,670 INFO monitors.py [line:104] execute command is ps -ef | grep 'ceph-mon' 
2017-06-13 09:50:53,909 INFO monitors.py [line:106] oot     35131     1  2 09:49 ?        00:00:01 /usr/local/bin/ceph-mon --cluster=ceph -i tahoe126 -f --setuser root --setgroup root
denali   41026 41025  0 09:50 ?        00:00:00 bash -c ps -ef | grep 'ceph-mon' 
denali   41028 41026  0 09:50 ?        00:00:00 grep ceph-mon

2017-06-13 09:50:53,909 INFO monitors.py [line:114] taheo126 is alrady started
2017-06-13 09:51:23,922 INFO cluster.py [line:211] execute command is sudo -i ceph -s
2017-06-13 09:51:24,332 INFO cluster.py [line:213]    cluster c1b31c6e-ebc7-4c97-98ec-d59964f99e42
     health HEALTH_OK
     monmap e3: 3 mons at {taheo125=192.168.40.125:6789/0,tahoe126=192.168.40.126:6789/0,tahoe127=192.168.40.127:6789/0}
            election epoch 60, quorum 0,1,2 taheo125,tahoe126,tahoe127
     osdmap e429: 18 osds: 18 up, 18 in
            flags sortbitwise,require_jewel_osds,require_kraken_osds
      pgmap v50637: 3216 pgs, 13 pools, 1498 GB data, 375 kobjects
            2055 GB used, 4134 GB / 6189 GB avail
                3216 active+clean
  client io 166 MB/s rd, 21457 op/s rd, 0 op/s wr
stdin: is not a tty

2017-06-13 09:51:24,333 INFO cluster.py [line:238] PG number is 3216
2017-06-13 09:51:24,333 INFO cluster.py [line:239] usefull PG number is 3216
2017-06-13 09:51:24,333 INFO TC56_kill_non_leader_mon.py [line:67] stop mon service on taheo126 in cluster successfully
2017-06-13 09:51:24,995 INFO monitors.py [line:126]    "quorum_leader_name": "taheo125",
stdin: is not a tty

2017-06-13 09:51:24,995 INFO monitors.py [line:129]    "quorum_leader_name": "taheo125",
2017-06-13 09:51:24,995 INFO TC56_kill_non_leader_mon.py [line:83] now the leader mon is taheo125
2017-06-13 09:51:24,995 INFO TC56_kill_non_leader_mon.py [line:85] the leader mon is not impacted
2017-06-13 09:51:25,225 INFO client.py [line:172] ['oot      33365      1  0 Jun12 ?        00:00:00 sudo -i fio -direct=1 -iodepth_batch_complete=1 -ioengine=rbd -clientname=client100 -pool=reliablityTestPool -rbdname=client100rbdImg2 -rw=randwrite -bs=8K -size=300G -norandommap=1 -randrepeat=0 -verify=md5 -verify_fatal=1 -verify_dump=1 -do_verify=1 -overwrite=1 -iodepth=256 -name=client100rbdImg2', 'root      33366  33365 84 Jun12 ?        01:46:55 fio -direct=1 -iodepth_batch_complete=1 -ioengine=rbd -clientname=client100 -pool=reliablityTestPool -rbdname=client100rbdImg2 -rw=randwrite -bs=8K -size=300G -norandommap=1 -randrepeat=0 -verify=md5 -verify_fatal=1 -verify_dump=1 -do_verify=1 -overwrite=1 -iodepth=256 -name=client100rbdImg2', 'root      33436      1  0 Jun12 ?        00:00:00 sudo -i fio -direct=1 -iodepth_batch_complete=1 -ioengine=rbd -clientname=client100 -pool=reliablityTestPool -rbdname=client100rbdImg3 -rw=randwrite -bs=8K -size=400G -norandommap=1 -randrepeat=0 -verify=md5 -verify_fatal=1 -verify_dump=1 -do_verify=1 -overwrite=1 -iodepth=256 -name=client100rbdImg3', 'root      33439  33436 99 Jun12 ?        02:06:46 fio -direct=1 -iodepth_batch_complete=1 -ioengine=rbd -clientname=client100 -pool=reliablityTestPool -rbdname=client100rbdImg3 -rw=randwrite -bs=8K -size=400G -norandommap=1 -randrepeat=0 -verify=md5 -verify_fatal=1 -verify_dump=1 -do_verify=1 -overwrite=1 -iodepth=256 -name=client100rbdImg3', 'root      33508      1  0 Jun12 ?        00:00:00 sudo -i fio -direct=1 -iodepth_batch_complete=1 -ioengine=rbd -clientname=client100 -pool=reliablityTestPool -rbdname=client100rbdImg4 -rw=randwrite -bs=8K -size=500G -norandommap=1 -randrepeat=0 -verify=md5 -verify_fatal=1 -verify_dump=1 -do_verify=1 -overwrite=1 -iodepth=256 -name=client100rbdImg4', 'root      33510  33508 99 Jun12 ?        02:30:08 fio -direct=1 -iodepth_batch_complete=1 -ioengine=rbd -clientname=client100 -pool=reliablityTestPool -rbdname=client100rbdImg4 -rw=randwrite -bs=8K -size=500G -norandommap=1 -randrepeat=0 -verify=md5 -verify_fatal=1 -verify_dump=1 -do_verify=1 -overwrite=1 -iodepth=256 -name=client100rbdImg4', 'denali    51793  51792  0 01:51 ?        00:00:00 bash -c sudo -i ps -ef | grep fio ', 'denali    51795  51793  0 01:51 ?        00:00:00 grep fio', 'stdin: is not a tty', '']
2017-06-13 09:51:25,225 INFO client.py [line:174] IO is running
2017-06-13 09:51:25,478 INFO monitors.py [line:45] ['oot     35131     1  2 09:49 ?        00:00:02 /usr/local/bin/ceph-mon --cluster=ceph -i tahoe126 -f --setuser root --setgroup root', 'denali   43811 43810  0 09:51 ?        00:00:00 bash -c ps -ef | grep ceph-mon', 'denali   43813 43811  0 09:51 ?        00:00:00 grep ceph-mon', '']
2017-06-13 09:51:25,478 INFO monitors.py [line:51] mon pid is 35131
2017-06-13 09:51:25,478 INFO monitors.py [line:92] execute command is sudo -i kill -9 35131 & sleep 3
2017-06-13 09:51:59,335 INFO monitors.py [line:126]    "quorum_leader_name": "taheo125",
stdin: is not a tty

2017-06-13 09:51:59,335 INFO monitors.py [line:129]    "quorum_leader_name": "taheo125",
2017-06-13 09:51:59,336 INFO TC56_kill_non_leader_mon.py [line:50] now deal the non-leader mon  taheo125
2017-06-13 09:51:59,336 INFO monitors.py [line:68] mon is  taheo126
2017-06-13 09:51:59,336 INFO monitors.py [line:69] execute command is sudo -i ceph-mon -i taheo126 & sleep 30
2017-06-13 09:52:29,522 ERROR monitors.py [line:75] Error when start mon taheo126
2017-06-13 09:52:29,522 ERROR monitors.py [line:76] sudo -i ceph-mon -i taheo126 & sleep 30
2017-06-13 09:52:29,522 ERROR monitors.py [line:77] tdin: is not a tty
monitor data directory at '/var/lib/ceph/mon/ceph-taheo126' does not exist: have you run 'mkfs'?

2017-06-13 09:52:29,522 INFO monitors.py [line:103] node is  taheo126
2017-06-13 09:52:29,522 INFO monitors.py [line:104] execute command is ps -ef | grep 'ceph-mon' 
2017-06-13 09:52:29,779 INFO monitors.py [line:106] enali   49811 49803  0 09:52 ?        00:00:00 bash -c ps -ef | grep 'ceph-mon' 
denali   49818 49811  0 09:52 ?        00:00:00 grep ceph-mon

2017-06-13 09:52:29,779 INFO monitors.py [line:110] taheo126 is not started, start again
2017-06-13 09:52:29,779 INFO monitors.py [line:68] mon is  taheo126
2017-06-13 09:52:29,780 INFO monitors.py [line:69] execute command is sudo -i ceph-mon -i taheo126 & sleep 30
2017-06-13 09:52:59,964 ERROR monitors.py [line:75] Error when start mon taheo126
2017-06-13 09:52:59,964 ERROR monitors.py [line:76] sudo -i ceph-mon -i taheo126 & sleep 30
2017-06-13 09:52:59,964 ERROR monitors.py [line:77] tdin: is not a tty
monitor data directory at '/var/lib/ceph/mon/ceph-taheo126' does not exist: have you run 'mkfs'?

2017-06-13 09:53:29,983 INFO cluster.py [line:211] execute command is sudo -i ceph -s
2017-06-13 09:53:33,378 INFO cluster.py [line:213]    cluster c1b31c6e-ebc7-4c97-98ec-d59964f99e42
     health HEALTH_WARN
            1 mons down, quorum 0,2 taheo125,tahoe127
     monmap e3: 3 mons at {taheo125=192.168.40.125:6789/0,tahoe126=192.168.40.126:6789/0,tahoe127=192.168.40.127:6789/0}
            election epoch 62, quorum 0,2 taheo125,tahoe127
     osdmap e429: 18 osds: 18 up, 18 in
            flags sortbitwise,require_jewel_osds,require_kraken_osds
      pgmap v50681: 3216 pgs, 13 pools, 1498 GB data, 375 kobjects
            2055 GB used, 4134 GB / 6189 GB avail
                3216 active+clean
  client io 151 MB/s rd, 19488 op/s rd, 0 op/s wr
stdin: is not a tty

2017-06-13 09:53:33,378 INFO cluster.py [line:238] PG number is 3216
2017-06-13 09:53:33,378 INFO cluster.py [line:239] usefull PG number is 3216
2017-06-13 09:53:33,378 INFO TC56_kill_non_leader_mon.py [line:67] stop mon service on taheo126 in cluster successfully
2017-06-13 09:53:34,023 INFO monitors.py [line:126]    "quorum_leader_name": "taheo125",
stdin: is not a tty

2017-06-13 09:53:34,023 INFO monitors.py [line:129]    "quorum_leader_name": "taheo125",
2017-06-13 09:53:34,023 INFO TC56_kill_non_leader_mon.py [line:83] now the leader mon is taheo125
2017-06-13 09:53:34,023 INFO TC56_kill_non_leader_mon.py [line:85] the leader mon is not impacted
2017-06-13 09:53:34,241 INFO client.py [line:172] ['oot      33365      1  0 Jun12 ?        00:00:00 sudo -i fio -direct=1 -iodepth_batch_complete=1 -ioengine=rbd -clientname=client100 -pool=reliablityTestPool -rbdname=client100rbdImg2 -rw=randwrite -bs=8K -size=300G -norandommap=1 -randrepeat=0 -verify=md5 -verify_fatal=1 -verify_dump=1 -do_verify=1 -overwrite=1 -iodepth=256 -name=client100rbdImg2', 'root      33366  33365 84 Jun12 ?        01:48:24 fio -direct=1 -iodepth_batch_complete=1 -ioengine=rbd -clientname=client100 -pool=reliablityTestPool -rbdname=client100rbdImg2 -rw=randwrite -bs=8K -size=300G -norandommap=1 -randrepeat=0 -verify=md5 -verify_fatal=1 -verify_dump=1 -do_verify=1 -overwrite=1 -iodepth=256 -name=client100rbdImg2', 'root      33436      1  0 Jun12 ?        00:00:00 sudo -i fio -direct=1 -iodepth_batch_complete=1 -ioengine=rbd -clientname=client100 -pool=reliablityTestPool -rbdname=client100rbdImg3 -rw=randwrite -bs=8K -size=400G -norandommap=1 -randrepeat=0 -verify=md5 -verify_fatal=1 -verify_dump=1 -do_verify=1 -overwrite=1 -iodepth=256 -name=client100rbdImg3', 'root      33439  33436 99 Jun12 ?        02:08:16 fio -direct=1 -iodepth_batch_complete=1 -ioengine=rbd -clientname=client100 -pool=reliablityTestPool -rbdname=client100rbdImg3 -rw=randwrite -bs=8K -size=400G -norandommap=1 -randrepeat=0 -verify=md5 -verify_fatal=1 -verify_dump=1 -do_verify=1 -overwrite=1 -iodepth=256 -name=client100rbdImg3', 'root      33508      1  0 Jun12 ?        00:00:00 sudo -i fio -direct=1 -iodepth_batch_complete=1 -ioengine=rbd -clientname=client100 -pool=reliablityTestPool -rbdname=client100rbdImg4 -rw=randwrite -bs=8K -size=500G -norandommap=1 -randrepeat=0 -verify=md5 -verify_fatal=1 -verify_dump=1 -do_verify=1 -overwrite=1 -iodepth=256 -name=client100rbdImg4', 'root      33510  33508 99 Jun12 ?        02:31:38 fio -direct=1 -iodepth_batch_complete=1 -ioengine=rbd -clientname=client100 -pool=reliablityTestPool -rbdname=client100rbdImg4 -rw=randwrite -bs=8K -size=500G -norandommap=1 -randrepeat=0 -verify=md5 -verify_fatal=1 -verify_dump=1 -do_verify=1 -overwrite=1 -iodepth=256 -name=client100rbdImg4', 'denali    57338  57337  0 01:53 ?        00:00:00 bash -c sudo -i ps -ef | grep fio ', 'denali    57340  57338  0 01:53 ?        00:00:00 grep fio', 'stdin: is not a tty', '']
2017-06-13 09:53:34,241 INFO client.py [line:174] IO is running
2017-06-13 09:53:34,482 INFO monitors.py [line:45] ['enali   55308 55294  0 09:53 ?        00:00:00 bash -c ps -ef | grep ceph-mon', 'denali   55311 55308  0 09:53 ?        00:00:00 grep ceph-mon', '']
2017-06-13 09:53:34,482 INFO monitors.py [line:51] mon pid is 35131
2017-06-13 09:53:34,483 INFO monitors.py [line:92] execute command is sudo -i kill -9 35131 & sleep 3
2017-06-13 09:54:08,269 INFO monitors.py [line:126]    "quorum_leader_name": "taheo125",
stdin: is not a tty

2017-06-13 09:54:08,269 INFO monitors.py [line:129]    "quorum_leader_name": "taheo125",
2017-06-13 09:54:08,269 INFO TC56_kill_non_leader_mon.py [line:50] now deal the non-leader mon  taheo125
2017-06-13 09:54:08,270 INFO monitors.py [line:68] mon is  taheo126
2017-06-13 09:54:08,270 INFO monitors.py [line:69] execute command is sudo -i ceph-mon -i taheo126 & sleep 30
2017-06-13 09:54:38,486 ERROR monitors.py [line:75] Error when start mon taheo126
2017-06-13 09:54:38,487 ERROR monitors.py [line:76] sudo -i ceph-mon -i taheo126 & sleep 30
2017-06-13 09:54:38,487 ERROR monitors.py [line:77] tdin: is not a tty
monitor data directory at '/var/lib/ceph/mon/ceph-taheo126' does not exist: have you run 'mkfs'?

2017-06-13 09:54:38,487 INFO monitors.py [line:103] node is  taheo126
2017-06-13 09:54:38,487 INFO monitors.py [line:104] execute command is ps -ef | grep 'ceph-mon' 
2017-06-13 09:54:38,733 INFO monitors.py [line:106] enali   60978 60976  0 09:54 ?        00:00:00 bash -c ps -ef | grep 'ceph-mon' 
denali   60980 60978  0 09:54 ?        00:00:00 grep ceph-mon

2017-06-13 09:54:38,733 INFO monitors.py [line:110] taheo126 is not started, start again
2017-06-13 09:54:38,734 INFO monitors.py [line:68] mon is  taheo126
2017-06-13 09:54:38,734 INFO monitors.py [line:69] execute command is sudo -i ceph-mon -i taheo126 & sleep 30
2017-06-13 09:55:09,055 ERROR monitors.py [line:75] Error when start mon taheo126
2017-06-13 09:55:09,055 ERROR monitors.py [line:76] sudo -i ceph-mon -i taheo126 & sleep 30
2017-06-13 09:55:09,056 ERROR monitors.py [line:77] tdin: is not a tty
monitor data directory at '/var/lib/ceph/mon/ceph-taheo126' does not exist: have you run 'mkfs'?

2017-06-13 09:55:39,079 INFO cluster.py [line:211] execute command is sudo -i ceph -s
2017-06-13 09:55:39,485 INFO cluster.py [line:213]    cluster c1b31c6e-ebc7-4c97-98ec-d59964f99e42
     health HEALTH_WARN
            1 mons down, quorum 0,2 taheo125,tahoe127
     monmap e3: 3 mons at {taheo125=192.168.40.125:6789/0,tahoe126=192.168.40.126:6789/0,tahoe127=192.168.40.127:6789/0}
            election epoch 62, quorum 0,2 taheo125,tahoe127
     osdmap e429: 18 osds: 18 up, 18 in
            flags sortbitwise,require_jewel_osds,require_kraken_osds
      pgmap v50728: 3216 pgs, 13 pools, 1498 GB data, 375 kobjects
            2055 GB used, 4134 GB / 6189 GB avail
                3216 active+clean
  client io 143 MB/s rd, 18381 op/s rd, 0 op/s wr
stdin: is not a tty

2017-06-13 09:55:39,485 INFO cluster.py [line:238] PG number is 3216
2017-06-13 09:55:39,485 INFO cluster.py [line:239] usefull PG number is 3216
2017-06-13 09:55:39,485 INFO TC56_kill_non_leader_mon.py [line:67] stop mon service on taheo126 in cluster successfully
2017-06-13 09:55:43,111 INFO monitors.py [line:126]    "quorum_leader_name": "taheo125",
stdin: is not a tty

2017-06-13 09:55:43,112 INFO monitors.py [line:129]    "quorum_leader_name": "taheo125",
2017-06-13 09:55:43,112 INFO TC56_kill_non_leader_mon.py [line:83] now the leader mon is taheo125
2017-06-13 09:55:43,112 INFO TC56_kill_non_leader_mon.py [line:85] the leader mon is not impacted
2017-06-13 09:55:43,367 INFO client.py [line:172] ['oot      33365      1  0 Jun12 ?        00:00:00 sudo -i fio -direct=1 -iodepth_batch_complete=1 -ioengine=rbd -clientname=client100 -pool=reliablityTestPool -rbdname=client100rbdImg2 -rw=randwrite -bs=8K -size=300G -norandommap=1 -randrepeat=0 -verify=md5 -verify_fatal=1 -verify_dump=1 -do_verify=1 -overwrite=1 -iodepth=256 -name=client100rbdImg2', 'root      33366  33365 84 Jun12 ?        01:49:55 fio -direct=1 -iodepth_batch_complete=1 -ioengine=rbd -clientname=client100 -pool=reliablityTestPool -rbdname=client100rbdImg2 -rw=randwrite -bs=8K -size=300G -norandommap=1 -randrepeat=0 -verify=md5 -verify_fatal=1 -verify_dump=1 -do_verify=1 -overwrite=1 -iodepth=256 -name=client100rbdImg2', 'root      33436      1  0 Jun12 ?        00:00:00 sudo -i fio -direct=1 -iodepth_batch_complete=1 -ioengine=rbd -clientname=client100 -pool=reliablityTestPool -rbdname=client100rbdImg3 -rw=randwrite -bs=8K -size=400G -norandommap=1 -randrepeat=0 -verify=md5 -verify_fatal=1 -verify_dump=1 -do_verify=1 -overwrite=1 -iodepth=256 -name=client100rbdImg3', 'root      33439  33436 99 Jun12 ?        02:09:46 fio -direct=1 -iodepth_batch_complete=1 -ioengine=rbd -clientname=client100 -pool=reliablityTestPool -rbdname=client100rbdImg3 -rw=randwrite -bs=8K -size=400G -norandommap=1 -randrepeat=0 -verify=md5 -verify_fatal=1 -verify_dump=1 -do_verify=1 -overwrite=1 -iodepth=256 -name=client100rbdImg3', 'root      33508      1  0 Jun12 ?        00:00:00 sudo -i fio -direct=1 -iodepth_batch_complete=1 -ioengine=rbd -clientname=client100 -pool=reliablityTestPool -rbdname=client100rbdImg4 -rw=randwrite -bs=8K -size=500G -norandommap=1 -randrepeat=0 -verify=md5 -verify_fatal=1 -verify_dump=1 -do_verify=1 -overwrite=1 -iodepth=256 -name=client100rbdImg4', 'root      33510  33508 99 Jun12 ?        02:33:09 fio -direct=1 -iodepth_batch_complete=1 -ioengine=rbd -clientname=client100 -pool=reliablityTestPool -rbdname=client100rbdImg4 -rw=randwrite -bs=8K -size=500G -norandommap=1 -randrepeat=0 -verify=md5 -verify_fatal=1 -verify_dump=1 -do_verify=1 -overwrite=1 -iodepth=256 -name=client100rbdImg4', 'denali    63140  63139  0 01:55 ?        00:00:00 bash -c sudo -i ps -ef | grep fio ', 'denali    63143  63140  0 01:55 ?        00:00:00 grep fio', 'stdin: is not a tty', '']
2017-06-13 09:55:43,367 INFO client.py [line:174] IO is running
2017-06-13 09:55:43,617 INFO monitors.py [line:45] ['enali   66523 66522  0 09:55 ?        00:00:00 bash -c ps -ef | grep ceph-mon', 'denali   66525 66523  0 09:55 ?        00:00:00 grep ceph-mon', '']
2017-06-13 09:55:43,617 INFO monitors.py [line:51] mon pid is 35131
2017-06-13 09:55:43,617 INFO monitors.py [line:92] execute command is sudo -i kill -9 35131 & sleep 3
2017-06-13 09:56:17,440 INFO monitors.py [line:126]    "quorum_leader_name": "taheo125",
stdin: is not a tty

2017-06-13 09:56:17,440 INFO monitors.py [line:129]    "quorum_leader_name": "taheo125",
2017-06-13 09:56:17,441 INFO TC56_kill_non_leader_mon.py [line:50] now deal the non-leader mon  taheo125
2017-06-13 09:56:17,441 INFO monitors.py [line:68] mon is  taheo126
2017-06-13 09:56:17,441 INFO monitors.py [line:69] execute command is sudo -i ceph-mon -i taheo126 & sleep 30
2017-06-13 09:56:47,628 ERROR monitors.py [line:75] Error when start mon taheo126
2017-06-13 09:56:47,628 ERROR monitors.py [line:76] sudo -i ceph-mon -i taheo126 & sleep 30
2017-06-13 09:56:47,628 ERROR monitors.py [line:77] tdin: is not a tty
monitor data directory at '/var/lib/ceph/mon/ceph-taheo126' does not exist: have you run 'mkfs'?

2017-06-13 09:56:47,628 INFO monitors.py [line:103] node is  taheo126
2017-06-13 09:56:47,628 INFO monitors.py [line:104] execute command is ps -ef | grep 'ceph-mon' 
2017-06-13 09:56:47,877 INFO monitors.py [line:106] enali   72175 72174  0 09:56 ?        00:00:00 bash -c ps -ef | grep 'ceph-mon' 
denali   72177 72175  0 09:56 ?        00:00:00 grep ceph-mon

2017-06-13 09:56:47,877 INFO monitors.py [line:110] taheo126 is not started, start again
2017-06-13 09:56:47,878 INFO monitors.py [line:68] mon is  taheo126
2017-06-13 09:56:47,878 INFO monitors.py [line:69] execute command is sudo -i ceph-mon -i taheo126 & sleep 30
2017-06-13 09:57:18,140 ERROR monitors.py [line:75] Error when start mon taheo126
2017-06-13 09:57:18,140 ERROR monitors.py [line:76] sudo -i ceph-mon -i taheo126 & sleep 30
2017-06-13 09:57:18,140 ERROR monitors.py [line:77] tdin: is not a tty
monitor data directory at '/var/lib/ceph/mon/ceph-taheo126' does not exist: have you run 'mkfs'?

2017-06-13 09:57:48,156 INFO cluster.py [line:211] execute command is sudo -i ceph -s
2017-06-13 09:57:51,526 INFO cluster.py [line:213]    cluster c1b31c6e-ebc7-4c97-98ec-d59964f99e42
     health HEALTH_WARN
            1 mons down, quorum 0,2 taheo125,tahoe127
     monmap e3: 3 mons at {taheo125=192.168.40.125:6789/0,tahoe126=192.168.40.126:6789/0,tahoe127=192.168.40.127:6789/0}
            election epoch 62, quorum 0,2 taheo125,tahoe127
     osdmap e429: 18 osds: 18 up, 18 in
            flags sortbitwise,require_jewel_osds,require_kraken_osds
      pgmap v50776: 3216 pgs, 13 pools, 1498 GB data, 375 kobjects
            2055 GB used, 4134 GB / 6189 GB avail
                3216 active+clean
  client io 161 MB/s rd, 20743 op/s rd, 0 op/s wr
stdin: is not a tty

2017-06-13 09:57:51,526 INFO cluster.py [line:238] PG number is 3216
2017-06-13 09:57:51,526 INFO cluster.py [line:239] usefull PG number is 3216
2017-06-13 09:57:51,526 INFO TC56_kill_non_leader_mon.py [line:67] stop mon service on taheo126 in cluster successfully
2017-06-13 09:57:52,167 INFO monitors.py [line:126]    "quorum_leader_name": "taheo125",
stdin: is not a tty

2017-06-13 09:57:52,167 INFO monitors.py [line:129]    "quorum_leader_name": "taheo125",
2017-06-13 09:57:52,167 INFO TC56_kill_non_leader_mon.py [line:83] now the leader mon is taheo125
2017-06-13 09:57:52,168 INFO TC56_kill_non_leader_mon.py [line:85] the leader mon is not impacted
2017-06-13 09:57:52,416 INFO client.py [line:172] ['oot      33365      1  0 Jun12 ?        00:00:00 sudo -i fio -direct=1 -iodepth_batch_complete=1 -ioengine=rbd -clientname=client100 -pool=reliablityTestPool -rbdname=client100rbdImg2 -rw=randwrite -bs=8K -size=300G -norandommap=1 -randrepeat=0 -verify=md5 -verify_fatal=1 -verify_dump=1 -do_verify=1 -overwrite=1 -iodepth=256 -name=client100rbdImg2', 'root      33366  33365 83 Jun12 ?        01:51:25 fio -direct=1 -iodepth_batch_complete=1 -ioengine=rbd -clientname=client100 -pool=reliablityTestPool -rbdname=client100rbdImg2 -rw=randwrite -bs=8K -size=300G -norandommap=1 -randrepeat=0 -verify=md5 -verify_fatal=1 -verify_dump=1 -do_verify=1 -overwrite=1 -iodepth=256 -name=client100rbdImg2', 'root      33436      1  0 Jun12 ?        00:00:00 sudo -i fio -direct=1 -iodepth_batch_complete=1 -ioengine=rbd -clientname=client100 -pool=reliablityTestPool -rbdname=client100rbdImg3 -rw=randwrite -bs=8K -size=400G -norandommap=1 -randrepeat=0 -verify=md5 -verify_fatal=1 -verify_dump=1 -do_verify=1 -overwrite=1 -iodepth=256 -name=client100rbdImg3', 'root      33439  33436 98 Jun12 ?        02:11:15 fio -direct=1 -iodepth_batch_complete=1 -ioengine=rbd -clientname=client100 -pool=reliablityTestPool -rbdname=client100rbdImg3 -rw=randwrite -bs=8K -size=400G -norandommap=1 -randrepeat=0 -verify=md5 -verify_fatal=1 -verify_dump=1 -do_verify=1 -overwrite=1 -iodepth=256 -name=client100rbdImg3', 'root      33508      1  0 Jun12 ?        00:00:00 sudo -i fio -direct=1 -iodepth_batch_complete=1 -ioengine=rbd -clientname=client100 -pool=reliablityTestPool -rbdname=client100rbdImg4 -rw=randwrite -bs=8K -size=500G -norandommap=1 -randrepeat=0 -verify=md5 -verify_fatal=1 -verify_dump=1 -do_verify=1 -overwrite=1 -iodepth=256 -name=client100rbdImg4', 'root      33510  33508 99 Jun12 ?        02:34:38 fio -direct=1 -iodepth_batch_complete=1 -ioengine=rbd -clientname=client100 -pool=reliablityTestPool -rbdname=client100rbdImg4 -rw=randwrite -bs=8K -size=500G -norandommap=1 -randrepeat=0 -verify=md5 -verify_fatal=1 -verify_dump=1 -do_verify=1 -overwrite=1 -iodepth=256 -name=client100rbdImg4', 'denali    68998  68997  0 01:57 ?        00:00:00 bash -c sudo -i ps -ef | grep fio ', 'denali    69000  68998  0 01:57 ?        00:00:00 grep fio', 'stdin: is not a tty', '']
2017-06-13 09:57:52,416 INFO client.py [line:174] IO is running
2017-06-13 09:57:52,668 INFO monitors.py [line:45] ['enali    5719  5718  0 09:57 ?        00:00:00 bash -c ps -ef | grep ceph-mon', 'denali    5721  5719  0 09:57 ?        00:00:00 grep ceph-mon', '']
2017-06-13 09:57:52,668 INFO monitors.py [line:51] mon pid is 35131
2017-06-13 09:57:52,668 INFO monitors.py [line:92] execute command is sudo -i kill -9 35131 & sleep 3
2017-06-13 09:58:26,475 INFO monitors.py [line:126]    "quorum_leader_name": "taheo125",
stdin: is not a tty

2017-06-13 09:58:26,475 INFO monitors.py [line:129]    "quorum_leader_name": "taheo125",
2017-06-13 09:58:26,475 INFO TC56_kill_non_leader_mon.py [line:50] now deal the non-leader mon  taheo125
2017-06-13 09:58:26,475 INFO monitors.py [line:68] mon is  taheo126
2017-06-13 09:58:26,475 INFO monitors.py [line:69] execute command is sudo -i ceph-mon -i taheo126 & sleep 30
2017-06-13 09:58:56,660 ERROR monitors.py [line:75] Error when start mon taheo126
2017-06-13 09:58:56,660 ERROR monitors.py [line:76] sudo -i ceph-mon -i taheo126 & sleep 30
2017-06-13 09:58:56,660 ERROR monitors.py [line:77] tdin: is not a tty
monitor data directory at '/var/lib/ceph/mon/ceph-taheo126' does not exist: have you run 'mkfs'?

2017-06-13 09:58:56,661 INFO monitors.py [line:103] node is  taheo126
2017-06-13 09:58:56,661 INFO monitors.py [line:104] execute command is ps -ef | grep 'ceph-mon' 
2017-06-13 09:58:56,940 INFO monitors.py [line:106] enali   11408 11404  0 09:59 ?        00:00:00 bash -c ps -ef | grep 'ceph-mon' 
denali   11410 11408  0 09:59 ?        00:00:00 grep ceph-mon

2017-06-13 09:58:56,940 INFO monitors.py [line:110] taheo126 is not started, start again
2017-06-13 09:58:56,940 INFO monitors.py [line:68] mon is  taheo126
2017-06-13 09:58:56,941 INFO monitors.py [line:69] execute command is sudo -i ceph-mon -i taheo126 & sleep 30
2017-06-13 09:59:27,125 ERROR monitors.py [line:75] Error when start mon taheo126
2017-06-13 09:59:27,125 ERROR monitors.py [line:76] sudo -i ceph-mon -i taheo126 & sleep 30
2017-06-13 09:59:27,125 ERROR monitors.py [line:77] tdin: is not a tty
monitor data directory at '/var/lib/ceph/mon/ceph-taheo126' does not exist: have you run 'mkfs'?

2017-06-13 09:59:57,140 INFO cluster.py [line:211] execute command is sudo -i ceph -s
2017-06-13 09:59:57,529 INFO cluster.py [line:213]    cluster c1b31c6e-ebc7-4c97-98ec-d59964f99e42
     health HEALTH_WARN
            1 mons down, quorum 0,2 taheo125,tahoe127
     monmap e3: 3 mons at {taheo125=192.168.40.125:6789/0,tahoe126=192.168.40.126:6789/0,tahoe127=192.168.40.127:6789/0}
            election epoch 62, quorum 0,2 taheo125,tahoe127
     osdmap e429: 18 osds: 18 up, 18 in
            flags sortbitwise,require_jewel_osds,require_kraken_osds
      pgmap v50822: 3216 pgs, 13 pools, 1498 GB data, 375 kobjects
            2055 GB used, 4134 GB / 6189 GB avail
                3216 active+clean
  client io 165 MB/s rd, 21179 op/s rd, 0 op/s wr
stdin: is not a tty

2017-06-13 09:59:57,529 INFO cluster.py [line:238] PG number is 3216
2017-06-13 09:59:57,530 INFO cluster.py [line:239] usefull PG number is 3216
2017-06-13 09:59:57,530 INFO TC56_kill_non_leader_mon.py [line:67] stop mon service on taheo126 in cluster successfully
2017-06-13 09:59:58,103 INFO monitors.py [line:126]    "quorum_leader_name": "taheo125",
stdin: is not a tty

2017-06-13 09:59:58,103 INFO monitors.py [line:129]    "quorum_leader_name": "taheo125",
2017-06-13 09:59:58,104 INFO TC56_kill_non_leader_mon.py [line:83] now the leader mon is taheo125
2017-06-13 09:59:58,104 INFO TC56_kill_non_leader_mon.py [line:85] the leader mon is not impacted
2017-06-13 09:59:58,352 INFO client.py [line:172] ['oot      33365      1  0 Jun12 ?        00:00:00 sudo -i fio -direct=1 -iodepth_batch_complete=1 -ioengine=rbd -clientname=client100 -pool=reliablityTestPool -rbdname=client100rbdImg2 -rw=randwrite -bs=8K -size=300G -norandommap=1 -randrepeat=0 -verify=md5 -verify_fatal=1 -verify_dump=1 -do_verify=1 -overwrite=1 -iodepth=256 -name=client100rbdImg2', 'root      33366  33365 83 Jun12 ?        01:52:53 fio -direct=1 -iodepth_batch_complete=1 -ioengine=rbd -clientname=client100 -pool=reliablityTestPool -rbdname=client100rbdImg2 -rw=randwrite -bs=8K -size=300G -norandommap=1 -randrepeat=0 -verify=md5 -verify_fatal=1 -verify_dump=1 -do_verify=1 -overwrite=1 -iodepth=256 -name=client100rbdImg2', 'root      33436      1  0 Jun12 ?        00:00:00 sudo -i fio -direct=1 -iodepth_batch_complete=1 -ioengine=rbd -clientname=client100 -pool=reliablityTestPool -rbdname=client100rbdImg3 -rw=randwrite -bs=8K -size=400G -norandommap=1 -randrepeat=0 -verify=md5 -verify_fatal=1 -verify_dump=1 -do_verify=1 -overwrite=1 -iodepth=256 -name=client100rbdImg3', 'root      33439  33436 98 Jun12 ?        02:12:43 fio -direct=1 -iodepth_batch_complete=1 -ioengine=rbd -clientname=client100 -pool=reliablityTestPool -rbdname=client100rbdImg3 -rw=randwrite -bs=8K -size=400G -norandommap=1 -randrepeat=0 -verify=md5 -verify_fatal=1 -verify_dump=1 -do_verify=1 -overwrite=1 -iodepth=256 -name=client100rbdImg3', 'root      33508      1  0 Jun12 ?        00:00:00 sudo -i fio -direct=1 -iodepth_batch_complete=1 -ioengine=rbd -clientname=client100 -pool=reliablityTestPool -rbdname=client100rbdImg4 -rw=randwrite -bs=8K -size=500G -norandommap=1 -randrepeat=0 -verify=md5 -verify_fatal=1 -verify_dump=1 -do_verify=1 -overwrite=1 -iodepth=256 -name=client100rbdImg4', 'root      33510  33508 99 Jun12 ?        02:36:06 fio -direct=1 -iodepth_batch_complete=1 -ioengine=rbd -clientname=client100 -pool=reliablityTestPool -rbdname=client100rbdImg4 -rw=randwrite -bs=8K -size=500G -norandommap=1 -randrepeat=0 -verify=md5 -verify_fatal=1 -verify_dump=1 -do_verify=1 -overwrite=1 -iodepth=256 -name=client100rbdImg4', 'denali    74690  74689  0 02:00 ?        00:00:00 bash -c sudo -i ps -ef | grep fio ', 'denali    74692  74690  0 02:00 ?        00:00:00 grep fio', 'stdin: is not a tty', '']
2017-06-13 09:59:58,353 INFO client.py [line:174] IO is running
2017-06-13 09:59:58,615 INFO monitors.py [line:45] ['enali   16722 16715  0 10:00 ?        00:00:00 bash -c ps -ef | grep ceph-mon', 'denali   16724 16722  0 10:00 ?        00:00:00 grep ceph-mon', '']
2017-06-13 09:59:58,615 INFO monitors.py [line:51] mon pid is 35131
2017-06-13 09:59:58,615 INFO monitors.py [line:92] execute command is sudo -i kill -9 35131 & sleep 3
2017-06-13 10:00:35,647 INFO monitors.py [line:126]    "quorum_leader_name": "taheo125",
stdin: is not a tty

2017-06-13 10:00:35,647 INFO monitors.py [line:129]    "quorum_leader_name": "taheo125",
2017-06-13 10:00:35,647 INFO TC56_kill_non_leader_mon.py [line:50] now deal the non-leader mon  taheo125
2017-06-13 10:00:35,648 INFO monitors.py [line:68] mon is  taheo126
2017-06-13 10:00:35,648 INFO monitors.py [line:69] execute command is sudo -i ceph-mon -i taheo126 & sleep 30
2017-06-13 10:01:05,800 ERROR monitors.py [line:75] Error when start mon taheo126
2017-06-13 10:01:05,800 ERROR monitors.py [line:76] sudo -i ceph-mon -i taheo126 & sleep 30
2017-06-13 10:01:05,801 ERROR monitors.py [line:77] tdin: is not a tty
monitor data directory at '/var/lib/ceph/mon/ceph-taheo126' does not exist: have you run 'mkfs'?

2017-06-13 10:01:05,801 INFO monitors.py [line:103] node is  taheo126
2017-06-13 10:01:05,801 INFO monitors.py [line:104] execute command is ps -ef | grep 'ceph-mon' 
2017-06-13 10:01:06,051 INFO monitors.py [line:106] enali   22942 22941  0 10:01 ?        00:00:00 bash -c ps -ef | grep 'ceph-mon' 
denali   22944 22942  0 10:01 ?        00:00:00 grep ceph-mon

2017-06-13 10:01:06,052 INFO monitors.py [line:110] taheo126 is not started, start again
2017-06-13 10:01:06,052 INFO monitors.py [line:68] mon is  taheo126
2017-06-13 10:01:06,052 INFO monitors.py [line:69] execute command is sudo -i ceph-mon -i taheo126 & sleep 30
2017-06-13 10:01:36,274 ERROR monitors.py [line:75] Error when start mon taheo126
2017-06-13 10:01:36,274 ERROR monitors.py [line:76] sudo -i ceph-mon -i taheo126 & sleep 30
2017-06-13 10:01:36,274 ERROR monitors.py [line:77] tdin: is not a tty
monitor data directory at '/var/lib/ceph/mon/ceph-taheo126' does not exist: have you run 'mkfs'?

2017-06-13 10:02:06,304 INFO cluster.py [line:211] execute command is sudo -i ceph -s
2017-06-13 10:02:09,698 INFO cluster.py [line:213]    cluster c1b31c6e-ebc7-4c97-98ec-d59964f99e42
     health HEALTH_WARN
            1 mons down, quorum 0,2 taheo125,tahoe127
     monmap e3: 3 mons at {taheo125=192.168.40.125:6789/0,tahoe126=192.168.40.126:6789/0,tahoe127=192.168.40.127:6789/0}
            election epoch 62, quorum 0,2 taheo125,tahoe127
     osdmap e429: 18 osds: 18 up, 18 in
            flags sortbitwise,require_jewel_osds,require_kraken_osds
      pgmap v50872: 3216 pgs, 13 pools, 1498 GB data, 375 kobjects
            2055 GB used, 4134 GB / 6189 GB avail
                3216 active+clean
  client io 171 MB/s rd, 21947 op/s rd, 0 op/s wr
stdin: is not a tty

2017-06-13 10:02:09,699 INFO cluster.py [line:238] PG number is 3216
2017-06-13 10:02:09,699 INFO cluster.py [line:239] usefull PG number is 3216
2017-06-13 10:02:09,699 INFO TC56_kill_non_leader_mon.py [line:67] stop mon service on taheo126 in cluster successfully
2017-06-13 10:02:10,314 INFO monitors.py [line:126]    "quorum_leader_name": "taheo125",
stdin: is not a tty

2017-06-13 10:02:10,315 INFO monitors.py [line:129]    "quorum_leader_name": "taheo125",
2017-06-13 10:02:10,315 INFO TC56_kill_non_leader_mon.py [line:83] now the leader mon is taheo125
2017-06-13 10:02:10,315 INFO TC56_kill_non_leader_mon.py [line:85] the leader mon is not impacted
2017-06-13 10:02:10,551 INFO client.py [line:172] ['oot      33365      1  0 Jun12 ?        00:00:00 sudo -i fio -direct=1 -iodepth_batch_complete=1 -ioengine=rbd -clientname=client100 -pool=reliablityTestPool -rbdname=client100rbdImg2 -rw=randwrite -bs=8K -size=300G -norandommap=1 -randrepeat=0 -verify=md5 -verify_fatal=1 -verify_dump=1 -do_verify=1 -overwrite=1 -iodepth=256 -name=client100rbdImg2', 'root      33366  33365 83 Jun12 ?        01:54:26 fio -direct=1 -iodepth_batch_complete=1 -ioengine=rbd -clientname=client100 -pool=reliablityTestPool -rbdname=client100rbdImg2 -rw=randwrite -bs=8K -size=300G -norandommap=1 -randrepeat=0 -verify=md5 -verify_fatal=1 -verify_dump=1 -do_verify=1 -overwrite=1 -iodepth=256 -name=client100rbdImg2', 'root      33436      1  0 Jun12 ?        00:00:00 sudo -i fio -direct=1 -iodepth_batch_complete=1 -ioengine=rbd -clientname=client100 -pool=reliablityTestPool -rbdname=client100rbdImg3 -rw=randwrite -bs=8K -size=400G -norandommap=1 -randrepeat=0 -verify=md5 -verify_fatal=1 -verify_dump=1 -do_verify=1 -overwrite=1 -iodepth=256 -name=client100rbdImg3', 'root      33439  33436 97 Jun12 ?        02:14:15 fio -direct=1 -iodepth_batch_complete=1 -ioengine=rbd -clientname=client100 -pool=reliablityTestPool -rbdname=client100rbdImg3 -rw=randwrite -bs=8K -size=400G -norandommap=1 -randrepeat=0 -verify=md5 -verify_fatal=1 -verify_dump=1 -do_verify=1 -overwrite=1 -iodepth=256 -name=client100rbdImg3', 'root      33508      1  0 Jun12 ?        00:00:00 sudo -i fio -direct=1 -iodepth_batch_complete=1 -ioengine=rbd -clientname=client100 -pool=reliablityTestPool -rbdname=client100rbdImg4 -rw=randwrite -bs=8K -size=500G -norandommap=1 -randrepeat=0 -verify=md5 -verify_fatal=1 -verify_dump=1 -do_verify=1 -overwrite=1 -iodepth=256 -name=client100rbdImg4', 'root      33510  33508 99 Jun12 ?        02:37:39 fio -direct=1 -iodepth_batch_complete=1 -ioengine=rbd -clientname=client100 -pool=reliablityTestPool -rbdname=client100rbdImg4 -rw=randwrite -bs=8K -size=500G -norandommap=1 -randrepeat=0 -verify=md5 -verify_fatal=1 -verify_dump=1 -do_verify=1 -overwrite=1 -iodepth=256 -name=client100rbdImg4', 'denali    80595  80594  0 02:02 ?        00:00:00 bash -c sudo -i ps -ef | grep fio ', 'denali    80598  80595  0 02:02 ?        00:00:00 grep fio', 'stdin: is not a tty', '']
2017-06-13 10:02:10,551 INFO client.py [line:174] IO is running
2017-06-13 10:02:10,802 INFO monitors.py [line:45] ['enali   28981 28977  0 10:02 ?        00:00:00 bash -c ps -ef | grep ceph-mon', 'denali   28984 28981  0 10:02 ?        00:00:00 grep ceph-mon', '']
2017-06-13 10:02:10,802 INFO monitors.py [line:51] mon pid is 35131
2017-06-13 10:02:10,802 INFO monitors.py [line:92] execute command is sudo -i kill -9 35131 & sleep 3
2017-06-13 10:02:47,659 INFO monitors.py [line:126]    "quorum_leader_name": "taheo125",
stdin: is not a tty

2017-06-13 10:02:47,660 INFO monitors.py [line:129]    "quorum_leader_name": "taheo125",
2017-06-13 10:02:47,660 INFO TC56_kill_non_leader_mon.py [line:50] now deal the non-leader mon  taheo125
2017-06-13 10:02:47,660 INFO monitors.py [line:68] mon is  taheo126
2017-06-13 10:02:47,660 INFO monitors.py [line:69] execute command is sudo -i ceph-mon -i taheo126 & sleep 30
2017-06-13 10:03:17,845 ERROR monitors.py [line:75] Error when start mon taheo126
2017-06-13 10:03:17,845 ERROR monitors.py [line:76] sudo -i ceph-mon -i taheo126 & sleep 30
2017-06-13 10:03:17,845 ERROR monitors.py [line:77] tdin: is not a tty
monitor data directory at '/var/lib/ceph/mon/ceph-taheo126' does not exist: have you run 'mkfs'?

2017-06-13 10:03:17,845 INFO monitors.py [line:103] node is  taheo126
2017-06-13 10:03:17,845 INFO monitors.py [line:104] execute command is ps -ef | grep 'ceph-mon' 
2017-06-13 10:03:18,096 INFO monitors.py [line:106] enali   34746 34745  0 10:03 ?        00:00:00 bash -c ps -ef | grep 'ceph-mon' 
denali   34748 34746  0 10:03 ?        00:00:00 grep ceph-mon

2017-06-13 10:03:18,096 INFO monitors.py [line:110] taheo126 is not started, start again
2017-06-13 10:03:18,096 INFO monitors.py [line:68] mon is  taheo126
2017-06-13 10:03:18,096 INFO monitors.py [line:69] execute command is sudo -i ceph-mon -i taheo126 & sleep 30
2017-06-13 10:03:48,313 ERROR monitors.py [line:75] Error when start mon taheo126
2017-06-13 10:03:48,314 ERROR monitors.py [line:76] sudo -i ceph-mon -i taheo126 & sleep 30
2017-06-13 10:03:48,314 ERROR monitors.py [line:77] tdin: is not a tty
monitor data directory at '/var/lib/ceph/mon/ceph-taheo126' does not exist: have you run 'mkfs'?

2017-06-13 10:04:18,342 INFO cluster.py [line:211] execute command is sudo -i ceph -s
2017-06-13 10:04:18,850 INFO cluster.py [line:213]    cluster c1b31c6e-ebc7-4c97-98ec-d59964f99e42
     health HEALTH_WARN
            1 mons down, quorum 0,2 taheo125,tahoe127
     monmap e3: 3 mons at {taheo125=192.168.40.125:6789/0,tahoe126=192.168.40.126:6789/0,tahoe127=192.168.40.127:6789/0}
            election epoch 62, quorum 0,2 taheo125,tahoe127
     osdmap e429: 18 osds: 18 up, 18 in
            flags sortbitwise,require_jewel_osds,require_kraken_osds
      pgmap v50919: 3216 pgs, 13 pools, 1498 GB data, 375 kobjects
            2055 GB used, 4134 GB / 6189 GB avail
                3216 active+clean
  client io 142 MB/s rd, 18326 op/s rd, 0 op/s wr
stdin: is not a tty

2017-06-13 10:04:18,850 INFO cluster.py [line:238] PG number is 3216
2017-06-13 10:04:18,850 INFO cluster.py [line:239] usefull PG number is 3216
2017-06-13 10:04:18,850 INFO TC56_kill_non_leader_mon.py [line:67] stop mon service on taheo126 in cluster successfully
2017-06-13 10:04:19,505 INFO monitors.py [line:126]    "quorum_leader_name": "taheo125",
stdin: is not a tty

2017-06-13 10:04:19,505 INFO monitors.py [line:129]    "quorum_leader_name": "taheo125",
2017-06-13 10:04:19,506 INFO TC56_kill_non_leader_mon.py [line:83] now the leader mon is taheo125
2017-06-13 10:04:19,506 INFO TC56_kill_non_leader_mon.py [line:85] the leader mon is not impacted
2017-06-13 10:04:19,726 INFO client.py [line:172] ['oot      33365      1  0 Jun12 ?        00:00:00 sudo -i fio -direct=1 -iodepth_batch_complete=1 -ioengine=rbd -clientname=client100 -pool=reliablityTestPool -rbdname=client100rbdImg2 -rw=randwrite -bs=8K -size=300G -norandommap=1 -randrepeat=0 -verify=md5 -verify_fatal=1 -verify_dump=1 -do_verify=1 -overwrite=1 -iodepth=256 -name=client100rbdImg2', 'root      33366  33365 83 Jun12 ?        01:55:57 fio -direct=1 -iodepth_batch_complete=1 -ioengine=rbd -clientname=client100 -pool=reliablityTestPool -rbdname=client100rbdImg2 -rw=randwrite -bs=8K -size=300G -norandommap=1 -randrepeat=0 -verify=md5 -verify_fatal=1 -verify_dump=1 -do_verify=1 -overwrite=1 -iodepth=256 -name=client100rbdImg2', 'root      33436      1  0 Jun12 ?        00:00:00 sudo -i fio -direct=1 -iodepth_batch_complete=1 -ioengine=rbd -clientname=client100 -pool=reliablityTestPool -rbdname=client100rbdImg3 -rw=randwrite -bs=8K -size=400G -norandommap=1 -randrepeat=0 -verify=md5 -verify_fatal=1 -verify_dump=1 -do_verify=1 -overwrite=1 -iodepth=256 -name=client100rbdImg3', 'root      33439  33436 97 Jun12 ?        02:15:45 fio -direct=1 -iodepth_batch_complete=1 -ioengine=rbd -clientname=client100 -pool=reliablityTestPool -rbdname=client100rbdImg3 -rw=randwrite -bs=8K -size=400G -norandommap=1 -randrepeat=0 -verify=md5 -verify_fatal=1 -verify_dump=1 -do_verify=1 -overwrite=1 -iodepth=256 -name=client100rbdImg3', 'root      33508      1  0 Jun12 ?        00:00:00 sudo -i fio -direct=1 -iodepth_batch_complete=1 -ioengine=rbd -clientname=client100 -pool=reliablityTestPool -rbdname=client100rbdImg4 -rw=randwrite -bs=8K -size=500G -norandommap=1 -randrepeat=0 -verify=md5 -verify_fatal=1 -verify_dump=1 -do_verify=1 -overwrite=1 -iodepth=256 -name=client100rbdImg4', 'root      33510  33508 99 Jun12 ?        02:39:09 fio -direct=1 -iodepth_batch_complete=1 -ioengine=rbd -clientname=client100 -pool=reliablityTestPool -rbdname=client100rbdImg4 -rw=randwrite -bs=8K -size=500G -norandommap=1 -randrepeat=0 -verify=md5 -verify_fatal=1 -verify_dump=1 -do_verify=1 -overwrite=1 -iodepth=256 -name=client100rbdImg4', 'denali    86676  86675  0 02:04 ?        00:00:00 bash -c sudo -i ps -ef | grep fio ', 'denali    86678  86676  0 02:04 ?        00:00:00 grep fio', 'stdin: is not a tty', '']
2017-06-13 10:04:19,726 INFO client.py [line:174] IO is running
2017-06-13 10:04:19,989 INFO monitors.py [line:45] ['enali   40375 40374  0 10:04 ?        00:00:00 bash -c ps -ef | grep ceph-mon', 'denali   40377 40375  0 10:04 ?        00:00:00 grep ceph-mon', '']
2017-06-13 10:04:19,989 INFO monitors.py [line:51] mon pid is 35131
2017-06-13 10:04:19,990 INFO monitors.py [line:92] execute command is sudo -i kill -9 35131 & sleep 3
2017-06-13 10:04:53,825 INFO monitors.py [line:126]    "quorum_leader_name": "taheo125",
stdin: is not a tty

2017-06-13 10:04:53,826 INFO monitors.py [line:129]    "quorum_leader_name": "taheo125",
2017-06-13 10:04:53,826 INFO TC56_kill_non_leader_mon.py [line:50] now deal the non-leader mon  taheo125
2017-06-13 10:04:53,826 INFO monitors.py [line:68] mon is  taheo126
2017-06-13 10:04:53,826 INFO monitors.py [line:69] execute command is sudo -i ceph-mon -i taheo126 & sleep 30
2017-06-13 10:05:24,044 ERROR monitors.py [line:75] Error when start mon taheo126
2017-06-13 10:05:24,044 ERROR monitors.py [line:76] sudo -i ceph-mon -i taheo126 & sleep 30
2017-06-13 10:05:24,045 ERROR monitors.py [line:77] tdin: is not a tty
monitor data directory at '/var/lib/ceph/mon/ceph-taheo126' does not exist: have you run 'mkfs'?

2017-06-13 10:05:24,045 INFO monitors.py [line:103] node is  taheo126
2017-06-13 10:05:24,045 INFO monitors.py [line:104] execute command is ps -ef | grep 'ceph-mon' 
2017-06-13 10:05:24,266 INFO monitors.py [line:106] enali   46420 46419  0 10:05 ?        00:00:00 bash -c ps -ef | grep 'ceph-mon' 
denali   46422 46420  0 10:05 ?        00:00:00 grep ceph-mon

2017-06-13 10:05:24,266 INFO monitors.py [line:110] taheo126 is not started, start again
2017-06-13 10:05:24,266 INFO monitors.py [line:68] mon is  taheo126
2017-06-13 10:05:24,266 INFO monitors.py [line:69] execute command is sudo -i ceph-mon -i taheo126 & sleep 30
2017-06-13 10:05:54,452 ERROR monitors.py [line:75] Error when start mon taheo126
2017-06-13 10:05:54,452 ERROR monitors.py [line:76] sudo -i ceph-mon -i taheo126 & sleep 30
2017-06-13 10:05:54,452 ERROR monitors.py [line:77] tdin: is not a tty
monitor data directory at '/var/lib/ceph/mon/ceph-taheo126' does not exist: have you run 'mkfs'?

2017-06-13 10:06:24,482 INFO cluster.py [line:211] execute command is sudo -i ceph -s
2017-06-13 10:06:24,861 INFO cluster.py [line:213]    cluster c1b31c6e-ebc7-4c97-98ec-d59964f99e42
     health HEALTH_WARN
            1 mons down, quorum 0,2 taheo125,tahoe127
     monmap e3: 3 mons at {taheo125=192.168.40.125:6789/0,tahoe126=192.168.40.126:6789/0,tahoe127=192.168.40.127:6789/0}
            election epoch 62, quorum 0,2 taheo125,tahoe127
     osdmap e429: 18 osds: 18 up, 18 in
            flags sortbitwise,require_jewel_osds,require_kraken_osds
      pgmap v50966: 3216 pgs, 13 pools, 1498 GB data, 375 kobjects
            2055 GB used, 4134 GB / 6189 GB avail
                3216 active+clean
  client io 148 MB/s rd, 19073 op/s rd, 0 op/s wr
stdin: is not a tty

2017-06-13 10:06:24,861 INFO cluster.py [line:238] PG number is 3216
2017-06-13 10:06:24,862 INFO cluster.py [line:239] usefull PG number is 3216
2017-06-13 10:06:24,862 INFO TC56_kill_non_leader_mon.py [line:67] stop mon service on taheo126 in cluster successfully
2017-06-13 10:06:28,456 INFO monitors.py [line:126]    "quorum_leader_name": "taheo125",
stdin: is not a tty

2017-06-13 10:06:28,457 INFO monitors.py [line:129]    "quorum_leader_name": "taheo125",
2017-06-13 10:06:28,457 INFO TC56_kill_non_leader_mon.py [line:83] now the leader mon is taheo125
2017-06-13 10:06:28,457 INFO TC56_kill_non_leader_mon.py [line:85] the leader mon is not impacted
2017-06-13 10:06:28,711 INFO client.py [line:172] ['oot      33365      1  0 Jun12 ?        00:00:00 sudo -i fio -direct=1 -iodepth_batch_complete=1 -ioengine=rbd -clientname=client100 -pool=reliablityTestPool -rbdname=client100rbdImg2 -rw=randwrite -bs=8K -size=300G -norandommap=1 -randrepeat=0 -verify=md5 -verify_fatal=1 -verify_dump=1 -do_verify=1 -overwrite=1 -iodepth=256 -name=client100rbdImg2', 'root      33366  33365 82 Jun12 ?        01:57:27 fio -direct=1 -iodepth_batch_complete=1 -ioengine=rbd -clientname=client100 -pool=reliablityTestPool -rbdname=client100rbdImg2 -rw=randwrite -bs=8K -size=300G -norandommap=1 -randrepeat=0 -verify=md5 -verify_fatal=1 -verify_dump=1 -do_verify=1 -overwrite=1 -iodepth=256 -name=client100rbdImg2', 'root      33436      1  0 Jun12 ?        00:00:00 sudo -i fio -direct=1 -iodepth_batch_complete=1 -ioengine=rbd -clientname=client100 -pool=reliablityTestPool -rbdname=client100rbdImg3 -rw=randwrite -bs=8K -size=400G -norandommap=1 -randrepeat=0 -verify=md5 -verify_fatal=1 -verify_dump=1 -do_verify=1 -overwrite=1 -iodepth=256 -name=client100rbdImg3', 'root      33439  33436 96 Jun12 ?        02:17:15 fio -direct=1 -iodepth_batch_complete=1 -ioengine=rbd -clientname=client100 -pool=reliablityTestPool -rbdname=client100rbdImg3 -rw=randwrite -bs=8K -size=400G -norandommap=1 -randrepeat=0 -verify=md5 -verify_fatal=1 -verify_dump=1 -do_verify=1 -overwrite=1 -iodepth=256 -name=client100rbdImg3', 'root      33508      1  0 Jun12 ?        00:00:00 sudo -i fio -direct=1 -iodepth_batch_complete=1 -ioengine=rbd -clientname=client100 -pool=reliablityTestPool -rbdname=client100rbdImg4 -rw=randwrite -bs=8K -size=500G -norandommap=1 -randrepeat=0 -verify=md5 -verify_fatal=1 -verify_dump=1 -do_verify=1 -overwrite=1 -iodepth=256 -name=client100rbdImg4', 'root      33510  33508 99 Jun12 ?        02:40:41 fio -direct=1 -iodepth_batch_complete=1 -ioengine=rbd -clientname=client100 -pool=reliablityTestPool -rbdname=client100rbdImg4 -rw=randwrite -bs=8K -size=500G -norandommap=1 -randrepeat=0 -verify=md5 -verify_fatal=1 -verify_dump=1 -do_verify=1 -overwrite=1 -iodepth=256 -name=client100rbdImg4', 'denali    92537  92532  0 02:06 ?        00:00:00 bash -c sudo -i ps -ef | grep fio ', 'denali    92540  92537  0 02:06 ?        00:00:00 grep fio', 'stdin: is not a tty', '']
2017-06-13 10:06:28,711 INFO client.py [line:174] IO is running
2017-06-13 10:06:28,907 INFO TC56_kill_non_leader_mon.py [line:95] case runs complete
